docs/topics/README.md

```diff
@@ -0,0 +1,34 @@
+# Topic Guides
+
+Introductions to all the key parts of AKS-Engine you’ll need to know.
+
+- [AAD integration Walkthrough](aad.md)
+- [Architecture](architecture.md)
+- [Cluster Definitions](clusterdefinitions.md) ([Chinese](clusterdefinitions.zh-CN.md))
+- [Extensions](extensions.md)
+- [Features](features.md)
+- [Using GPUs with Kubernetes](gpu.md)
+- [Running Kubernetes in a hybrid environment](hybrid-environment.md)
+- [For Kubernetes Developers](kubernetes-developers.md)
+- [Kubernetes Walkthrough](kubernetes-walkthrough.md)
+- [Monitoring Kubernetes Clusters](monitoring.md)
+- [Scaling Kubernetes Clusters](scaling.md)
+- [Service Principals](service-principals.md)
+- [More on Windows and Kubernetes](windows-and-kubernetes.md)
+- [Kubernetes Windows Walkthrough](windows.md)
+
+## Community Material
+
+This material is external to the core documentation, but provide valuable pieces of information related to AKS-Engine thanks to the many community members.
+
+If you're new to AKS-Engine, adding snippets from these pieces into the core documentation is a great way to get started... Hint hint. ;)
+
+- [Getting started with the ACS Engine to deploy Kubernetes in Azure](http://starkfell.github.io/getting-started-with-using-the-acs-engine-to-deploy-k8s-in-azure/)
+
+## Additional Kubernetes Resources
+
+Here are recommended links to learn more about Kubernetes:
+
+- [Kubernetes Bootcamp](https://kubernetesbootcamp.github.io/kubernetes-bootcamp/index.html) - shows you how to deploy, scale, update, and debug containerized applications.

```

Comment:
nit: `some recommended` ?

Reply:
the subtitle comes directly from the landing page of the bootcamp. This was also pulled elsewhere in the docs; I'm just shifting things around.